src/http/server/handlers/static_file_handler.cr

```diff
@@ -115,10 +115,7 @@ class HTTP::StaticFileHandler
   end
 
   private def redirect_to(context, url)
```

Comment:
I think this method call should be inlined now. It's done only once, and it's exactly one line now.

Reply:
I'd like to avoid breaking `StaticFileHandler`'s internal API, even if it's an undocumented private method. This class if often inherited from in web frameworks or apps which use its private methods to override specific behaviour. (Not sure if that's actually the case with `redirect_to`, I didn't check)
But it's cheap to keep this method around. It should be inlined in codegen, so there is no performance cost.